src/Collections-Arithmetic-Tests/CollectionArithmeticTests.class.st

```diff
@@ -0,0 +1,44 @@
+Class {
+	#name : #CollectionArithmeticTests,
+	#superclass : #TestCase,
+	#category : #'Collections-Arithmetic-Tests'
+}
+
+{ #category : #tests }
+CollectionArithmeticTests >> testAverage [
+	| collection |
+	collection := #(1 2 3).
+	self assert: collection average equals: 2
+]
+
+{ #category : #tests }
+CollectionArithmeticTests >> testAverageIfEmpty [
+	| collection |
+	collection := #(1 2 3 4).
+	self assert: (collection averageIfEmpty: [ 0 ]) equals: 2.5
```

Comment:
I would also add 

```Smalltalk
self assert: (#() averageIfEmpty: [ 0 ]) equals: 0
```

Either here or in testAverageIfEmptyWithEmptyCollection since it will be the main use-case IMO.

Reply:
Test case added